proto/fundraising/fundraising.proto

```diff
@@ -239,11 +242,11 @@ message Bid {
   string bidder = 2;
 
   // id specifies an index of a bid for the bidder
-  uint64 id = 3;
+  uint64 bid_id = 3;
```

Comment:
Since id is already inside `Bid` struct, I think it can just be `id`.

Reply:
I updated based on the comments 1,2,3 above.
For adding NumWinningBidders to CreateFixedPriceAuction of keeper/auction.go, I added NumWinningBidders = 0.